rn-cli.config.js

```diff
@@ -1,11 +1,18 @@
 const blacklist = require('metro-config/src/defaults/blacklist')
 const path = require('path')
+const glob = require('glob-to-regexp')
 
 const getBlacklistRE = () => {
-  // delete __tests__ from the default blacklist
-  const defaultPattern = blacklist()
+  // ignore dist/, dev/
+  const defaultPattern = blacklist([
+    glob(`${path.resolve(__dirname, '..')}/dist/*`),
+    glob(`${path.resolve(__dirname, '..')}/dev/*`),
+    glob(`${path.resolve(__dirname, '..')}/example/*`),
```

Comment:
Hmm I am not sure if this is what you want but to ignore all nested files I think you should use
```
glob(`${path.resolve(__dirname, '..')}/example/**`, { globstars: true })
```

Reply:
my bad. it seems that this fix is still broken :( … and it's not because of this, because Metro trips up on package.json file which is directly in /dev/, for example. But you're right, i'll change it later.